scripts/pagerduty_notify.rb

```diff
@@ -82,6 +86,16 @@ def check_arguments(arglist)
 ap output_data if(ENV['debug'])
 #JSON.dump(output_data)
 
+disable_flag=false
+
+if ENV['disable_communications']
+  disable_flag=$store.substitute_string(ENV['disable_communications'])
+  if disable_flag.downcase=='yes' or disable_flag.downcase=='true' or disable_flag.downcase=='disable'
```

Comment:
un-necessary from a strict point of view... yes. horrible... well arguably it makes it more usable so I wouldn't use that term.